src/Helpers/index.js

```diff
@@ -8,19 +8,57 @@ import mapboxgl from 'Lib';
  */
 const CIRCLE_POINTS_CONFIG = 64;
 
+/**
+ * Array of possible units
+ * @type {Array}
+ */
+const UNITS = ["kilometers", "meters", "miles", "feet"];
+
+/**
+ * Convert the radius into the unit given
+ * @param {int} radius unit 
+ * @param {string} unit to convert to if necessary
+ */
+export function convertRadiusUnit(radius, unit) {
+    let convertedRadius = radius;
+    let convertedUnit = unit;
+
+    if(!Number.isInteger(radius)) {
+        global.console.error('Radius is not an integer');
```

Comment:
I asked myself about it. But in the end react should throw the error because we have the definition of radius in the proptypes: `radius: PropTypes.number.isRequired`.
so this test is not needed.